systemd/Dockerfile.fedora.tpl

```diff
@@ -0,0 +1,47 @@
+FROM #{FROM}
+
+RUN rpm --rebuilddb && yum install -y \
+        ca-certificates tar \
+        systemd \
+    && yum clean all
+
+# Install Systemd
+
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
```

Comment:
Why do we delete all these files? We should touch /lib folders, the correct way to do it is via symlinks in /etc. `systemd disable` puts them there automatically